SWIPs/swip-honey_to_money.md

```diff
@@ -7,51 +7,66 @@ Type: Standards track
+As nobody benefits from price imbalances, both the sender and the receiver of a payment are incentivized to and can apply a price which is not per their clock, but per the clock of their counterparty when they expect their counterparty to query or have queried the oracle. To assist in this, a payload might be sent with the transaction, specifying the clock with which the oracle price was applied.
+* An on-chain oracle *might* give a different answer to two nodes when queried at the same time if there exists a (temporary) fork in the blockchain. Again: the logic applies here about the severity of such accounting imbalances. If this appears to be severe, any oracle price-update might be only considered valid if it happened a sufficient amount of time in the past (usually 12 blocks in Ethereum)
 
 ## Backwards Compatibility 
-This SWIP is backward compatible as long as the price oracle quotes the same prices as is currently hard-coded in non-upgraded Swarm nodes. It is up to the owner of the `MsgToHoney` oracle to ensure that prices are not updated too much while not all running nodes are on the new Swarm. Currently, Swarm is not running a Swap-enabled live test net, so we don't expect any problems with this SWIP if it is implemented before Swarm will go live with incentivization. 
+<!--All SWIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The SWIP must explain how the author proposes to deal with these incompatibilities. SWIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+Currently, the Swarm is not running with a live test net for settling prices, so no problems are expected with this SWIP if it is implemented before the Swarm will go live with price-incentivization. 
```

Comment:
Is it not "setting" prices? Settling would be for imbalances, or cheques, no?

Reply:
It is "setting" indeed. Will update!